src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/SdkRuntimePacksTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.DotNet.SourceBuild.SmokeTests
+{
+    public class SdkRuntimePacksTests : SmokeTests
+    {
+        private static readonly string _nuGetCacheFolder = "nuget-cache";
+
+        // `dotnet new webapp` will have dependency on the two runtime packs that are distributed with the SDK:
```

Comment:
This could also be a really easy sanity check - we could simply compare what was restored with what is in the SDK to make sure that we are not missing a pack that was added later. 

My understanding is that new packs are not added often so I did not include it here